dags/utils/tbv.py

```diff
@@ -17,7 +17,7 @@ def tbv_envvar(klass, options, other={}):
     :returns: a dictionary that contains properly prefixed class and options
     """
     prefixed_options = {
-        "TBV_{}".format(key.upper().replace("-", "_")): value
+        "TBV_{}".format(key.replace("-", "_")): value
```

Comment:
@acmiyaguchi I am concerned that this change will lag behind removing `.lower()` in `jobs/telemetry_batch_view.py`, which would cause jobs to fail. Could that happen?

Reply:
This is regards to the github repository not updating the raw file at the master location? I think it's a somewhat rare occurrence, but maybe separating this change into a separate patch would be a good idea.

What was the reasoning behind dropping `upper` in the first place?

Reply:
we dropped upper because we are preserving case now, because we have camel case args to pass. my concern is not that github will be slow. my concern is that airflow will not get the latest utils/tbv.py as fast as it gets the latest jobs/telemetry_batch_view.py for other jobs already using `jobs/telemetry_batch_view.py`, thus sending it upper cased env vars that become upper cased  args, and failing a dag run.

Reply:
I think the autodeploy mechanism will also update the utils code. We direcly import the tbv module into all of the dags, which means that the jobs encoded in the dags and the utility python code should get updated at the same time.

I'm also a little wary about preserving case in the environment since the convention is to have them all uppercased, but it looks like it should be fine. Maybe something to think about is an alternative way of passing context to the operator.